doc/lsst.ap.pipe/pipeline-with-fakes.rst

```diff
@@ -0,0 +1,194 @@
+Camera-Specific Configurations
+------------------------------
+
+Each camera has its own ApPipeWithFakes configuration that imports the ingredients file. For example:
+
+.. code-block:: yaml
+
+   instrument: lsst.obs.decam.DarkEnergyCamera
+   imports:
+     - location: $AP_PIPE_DIR/pipelines/_ingredients/ApPipeWithFakes.yaml
+
+   parameters:
+     coaddName: goodSeeing
+     apdb_config: <path_to_apdb_config>
+
+The available camera-specific configurations are:
+
+* ``pipelines/DECam/ApPipeWithFakes.yaml``
+* ``pipelines/HSC/ApPipeWithFakes.yaml``
+* ``pipelines/LATISS/ApPipeWithFakes.yaml``
+* ``pipelines/LSSTCam/ApPipeWithFakes.yaml``
+* ``pipelines/LSSTCam-imSim/ApPipeWithFakes.yaml``
+* ``pipelines/LSSTComCam/ApPipeWithFakes.yaml``
+
+
+Build-Time Pipeline Generation
+==============================
+
+The `ApPipeWithFakes.yaml` ingredients file is **automatically generated** during package compilation,
+not manually maintained. This process is controlled by the ``SConstruct`` file at the package root.
+
+SConstruct Configuration
+------------------------
+
+The ``SConstruct`` file defines the build process for generating the ApPipeWithFakes pipeline, this build process:
+
+1. **Takes the base ApPipe pipeline** (``ApPipe.yaml``) as input
+2. **Uses the ``make_injection_pipeline`` command** to automatically generate the fakes version
+3. **Adds injection-specific tasks** from ``ApPipePostInjectedTasks.yaml``
+4. **Applies systematic transformations**:
+
+   * Prefixes task names with ``fakes_`` to create parallel processing paths
+   * Configures injection parameters (PSF, photometry, WCS settings)
+   * Sets up proper data connections between injected and standard processing
+   * Creates analysis tasks for fake source performance evaluation
+
+5. **Ensures proper build dependencies**: The generated file depends on the version target and is required for tests
+
+Key Build Parameters
+~~~~~~~~~~~~~~~~~~~~
+
+* **Target exposure**: ``preliminary_visit_image`` - the image type to inject sources into
+* **Subsets processed**: ``apPipe`` and ``prompt`` - which pipeline subsets get fakes versions
+* **Configuration overrides**: Sets injection task defaults for PSF, photometry, and WCS handling
+* **APDB configuration**: Uses placeholder config that must be user-specified at runtime
+
+Why Generate at Build Time?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This approach provides several advantages:
+
+* **Consistency**: Ensures the fakes pipeline stays synchronized with the base pipeline
+* **Maintenance**: Automatically incorporates changes to the base pipeline without manual updates
+* **Flexibility**: Allows systematic application of injection patterns across different pipeline subsets
+* **Testing**: Ensures the fakes pipeline is always available for validation and testing
+
+The generated ``ApPipeWithFakes.yaml`` file should **not** be manually edited, as it will be
+overwritten during the next package build. Instead, modifications should be made to:
```

Comment:
Again, this warning does no good buried in what is otherwise a concept-level document.

Reply:
I only kept a clarification on what to really modify if needed.